.github/workflows/ci.yml

```diff
@@ -79,7 +79,7 @@ jobs:
         shell: bash
         run: |
           sudo apt update
-          sudo apt install libpulse-dev libegl1-mesa libopengl0
+          sudo apt install libpulse-dev libegl1-mesa libopengl0 gstreamer1.0-plugins-base gstreamer1.0-gl
```

Comment:
I think this is needed for `test_multimedia.py` but I think no changes have been done to that test file. Maybe this is needed for some other test? If not then I think we should revert this change

Reply:
this is needed for the pyqt6 wheels